internal/xds/cache/snapshotcache.go

```diff
@@ -55,7 +56,7 @@ type snapshotCache struct {
 	streamIDNodeInfo nodeInfoMap
 	snapshotVersion  int64
 	lastSnapshot     snapshotMap
-	log              *LogrWrapper
+	log              *zap.SugaredLogger
```

Comment:
curious, why ?

Reply:
LogrWrapper and zap.SugaredLogger have same api like `Debugf` ,`Infof` etc.

We.can use `zap.SugaredLogger` direct ly